src/helpers.ts

```diff
@@ -99,26 +99,29 @@ const prioritizeBldgsWithUnits = (
   b: RelatedProperty,
   unitsNeeded: number
 ) => {
-  if (a.acris_docs.length !== b.acris_docs.length)
-    return a.acris_docs.length ? -1 : 1;
-
   if (a.match_ownername !== b.match_ownername)
     return a.match_ownername ? -1 : 1;
 
   if (a.match_multidoc !== b.match_multidoc) return a.match_multidoc ? -1 : 1;
 
+  if (a.party_name_match !== b.party_name_match)
+    return a.party_name_match ? -1 : 1;
+
   if (a.wow_match_name !== b.wow_match_name) return a.wow_match_name ? -1 : 1;
 
   if (a.wow_match_bizaddr_unit !== b.wow_match_bizaddr_unit)
     return a.wow_match_bizaddr_unit ? -1 : 1;
 
+  if (a.acris_docs.length !== b.acris_docs.length)
+    return a.acris_docs.length ? -1 : 1;
+
   if (
     unitsNeeded > 0 &&
     a.unitsres >= unitsNeeded !== b.unitsres >= unitsNeeded
   )
     return a.unitsres >= unitsNeeded ? -1 : 1;
 
-  return b.distance_ft - a.distance_ft;
+  return a.distance_ft - b.distance_ft;
```

Comment:
why did you switch these values?

Reply:
I did some double checking of the final order and it seemed like I had it backwards before